lib/net.js

```diff
@@ -32,6 +32,7 @@ class Server extends net.Server {
     return new sock.Transport(socket);
   }
 }
+
```

Comment:
I didn't put this line back then to signify that it is directly related to the class above and not a standalone statement, not sure if we should add it.